### ISSUE TITLE
Get ready for new NCBI Taxonomy

### ISSUE THREAD

**Issue by crosenth** [fhcrc/taxtastic#171]
New taxonomy coming soon: 

https://ncbiinsights.ncbi.nlm.nih.gov/2024/06/04/changes-ncbi-taxonomy-classifications/

`taxit new_database` has been added to the GHA test workflow and subscribed the NCBI Insights blog for updates to come

**Comment by nach00gar** [comment]
Last night, files from ncbi were updated adding the domain rank. An update is required.

https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/

**Comment by crosenth** [comment]
Thank you for the heads up!

**Comment by crosenth** [comment]
@dhoogest: https://ncbiinsights.ncbi.nlm.nih.gov/2025/02/27/new-ranks-ncbi-taxonomy/

**Comment by crosenth** [comment]
@nach00gar  - v0.11.0 is out and should close this Issue, let me know

**Comment by crosenth** [comment]
New celllular_root and acellular_root

https://ncbiinsights.ncbi.nlm.nih.gov/2025/02/27/new-ranks-ncbi-taxonomy/#more-14432

https://github.com/fhcrc/taxtastic/compare/v0.11.0...v0.11.1

**Comment by dhoogest** [comment]
@yeemey yep there are a few nodes with that rank, which doesn't appear in the 'ranks' table (see related https://github.com/fhcrc/taxtastic/issues/175). It's possible NCBI will self-resolve, but this does re-raise question about how to make taxtastic more resilient during change periods
